ui/src/features/Editor/useCanvasCopyPaste.ts

```diff
@@ -27,31 +30,14 @@ export default ({
   handleEdgesAdd: (newEdges: Edge[]) => void;
 }) => {
   const { copy, paste } = useCopyPaste();
+  const { toast } = useToast();
+  const t = useT();
+  const [hasItemsToPaste, setHasItemsToPaste] = useState<boolean>(false);
```

Comment:
Sorry back to this again. I want you to remove this, which on a refresh would become false again despite the clipboard persisting, and instead where you are using hasItemsToPaste, query the indexedDB and use whether there is a value there or not to disable/hide the paste button